src/internal/components/live-region/dynamic-aria-live.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
should this be a separate component? 90% of it is duplicated from `live-region`, can throttling be added to the already existing component? 

if not I suggest to export the duplicated functionality from one component and use it in the other